pkg/k8s/init.go

```diff
@@ -245,6 +245,9 @@ func WaitForNodeInformation() error {
 
 		node.SetLabels(n.Labels)
 
+		node.SetK8sExternalIPv4(n.GetExternalIP(false))
+		node.SetK8sExternalIPv6(n.GetExternalIP(true))
```

Comment:
I didn't do `if option.Config.EnableIPv4 / if option.Config.EnableIPv6` check here because of the comment on line 232.

https://github.com/cilium/cilium/blob/0c760d8c5b4437e1017eb0ef9c0a74f84e68a6d4/pkg/k8s/init.go#L232-L237

Reply:
Should I still add it?

Reply:
In that case, maybe not. I presume that these external IPs would be suitable for use from clustermesh given that the requirement is they're publicly routable (though obviously this code is not making such a change as yet).